core/src/main/java/cucumber/runner/TimeService.java

```diff
@@ -2,12 +2,19 @@
 
 public interface TimeService {
     long time();
+    long timeStampMillis();
 
     TimeService SYSTEM = new TimeService() {
         @Override
         public long time() {
             return System.nanoTime();
         }
+
+        @Override
+        public long timeStampMillis()
```

Comment:
K&R brackets. You can probably configure this in your IDE on a project basis.

Reply:
Really sorry about that. Updated.